_config/config.yml

```diff
@@ -1,6 +1,12 @@
 LeftAndMain:
   extensions:
     - LeftAndMainPageIconsExtension
+ContentController:
```

Comment:
Okay sure. I was focused on changing as little as possible but no harm in sneaking in an improvement or two I suppose! So applying it to Controller means I don't need to specify ContentController, Security or ModelAsController as they all extend Controller. So it will just be on Controller and Form then, yes?